pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -743,6 +868,13 @@ func (r *ReconcileBareMetalHost) manageHostPower(prov provisioner.Provisioner, i
 	}
 
 	if provResult.Dirty {
+		info.postSaveCallbacks = append(info.postSaveCallbacks, func() {
+			if info.host.Spec.Online {
+				powerChangeAttempts.WithLabelValues(info.host.Name, "on").Inc()
+			} else {
+				powerChangeAttempts.WithLabelValues(info.host.Name, "off").Inc()
+			}
+		})
```

Comment:
This could still end up getting called multiple times for each power state change. What I think we really want to log is when the Ironic provisioner ended up having to call `changePower()`, but there's currently no way of determining that from the data returned by the `PowerOn()`/`PowerOff()` APIs.

Reply:
The idea I had with this metric was that we shouldn't ever need to power the server on/off in this context. So, having extra counts when we do would be OK, because as long as we don't miss any then we will have little spikes in the graph to trigger an alarm.